src/components/RecipeList.vue

```diff
@@ -75,6 +75,8 @@ export default {
       return this.items.length;
     },
     paginatedItems() {
+      let url = ( new URL(window.location.href).searchParams.get("page"));
+      if(url != null ? this.pageNumber=url-1 : this.pageNumber=0);
```

Comment:
Can you get the query param through `$route`?
```
this.$route.query.page
```

Reply:
Sure, I will have a look when i get home on my laptop :)

Reply:
Used  
> this.$route.query.page 
 to get query parameter